src/validation/fields.ts

```diff
@@ -133,3 +134,21 @@ export const activeMfaType = Joi.alternatives()
   .description(
     'Multi-factor authentication type. A null value deactivates MFA'
   );
+
+export const phoneNumber = Joi.string()
+  .required()
+  .custom((value: string) => {
+    // * Replace '00' by '+'
+    if (value.startsWith('00')) {
+      value = value.replace('00', '+');
+    }
```

Comment:
I think it makes sense to use `+` instead of `00`. The formatting is called [E.164](https://en.wikipedia.org/wiki/E.164) and the format that is [being used at Twilio](https://support.twilio.com/hc/en-us/articles/360050891214-Getting-Started-with-the-Twilio-Lookup-API#:~:text=This%20response%20indicates%20the%20phone%20number%20is%20valid%2C%20but%20not%20necessarily%20in%20service.%20The%20phone_number%20value%20is%20presented%20in%C2%A0E.164%20formatting%2C%20which%20should%20be%20used%20across%20Twilio%E2%80%99s%20voice%20and%20messaging%20APIs.).

But, given this is an opinionated decision from our side. Is there anything else we should consider here?